src/demos/main-hero.css

```diff
@@ -1,23 +1,86 @@
 .main-hero {
   clip-path: url(#myClipPath);
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  background: white;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 20px;
+}
+
+.main-hero-content {
+  /*   position: absolute;
+  top: 50%;
+  left: 20px;
+  transform: translateY(-50%);
+  z-index: 2;
+  color: black; */
   width: 100%;
-  height: 400px;
 }
-.container {
+
+.text-container {
+  position: absolute;
+  top: 25%;
+  max-width: 40vw;
+  padding: 0 20px 0 0;
+  /* flex: 1;
   display: flex;
+  padding: 20px;
   flex-direction: column;
-  align-items: center;
-  justify-content: flex-start;
-  text-align: center;
-  background-color: black;
-  height: 100vh;
+  max-width: 600px;
+  margin-right: 20px; */
+}
+.text-container h2 {
+  margin-bottom: 0;
+}
+.text-container p {
+  margin-top: 20px;
 }
-
 .isClipped {
   clip-path: url(#svgClip);
 }
+.image-container {
+  flex: 1;
+  display: flex;
+  justify-content: flex-end;
+  order: 1;
+  margin-bottom: 20px;
+}
+.image-container svg {
+  max-width: 100%;
+  height: auto;
+}
+
+@media screen and (max-width: 960px) {
```

Comment:
We want to create a responsive experience with as few media queries as possible. try to make the SVG responsive and auto-adjust to the view window. look at the most up-to-date pull of media module to get an idea of this. and reach out if you want me to go over it :D